components/centraldashboard/Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN BUILDARCH="$(dpkg --print-architecture)" &&  npm rebuild && \
 # Step 2: Packages assets for serving
 FROM node:12.22.12-alpine AS serve
 
+ARG kubeflowversion
+ARG commit
+ENV BUILD_VERSION=$kubeflowversion
+ENV BUILD_COMMIT=$commit
```

Comment:
Looks like it's the only way to use ARG in different stages. https://github.com/moby/moby/issues/37345#issuecomment-698633259
I don't think `BUILD_VERSION` and `BUILD_COMMIT` are necessary for the build stage but I'm not sure. It so, I can update the PR to remove it.